pkg/controller/container-runtime-config/helpers.go

```diff
@@ -515,6 +531,10 @@ func updatePolicyJSON(data []byte, internalBlocked, internalAllowed []string, re
 			signature.NewPRReject(),
 		}
 		for _, reg := range internalAllowed {
+			if clusterScopePolicies[reg] != nil {
+				// do not set "insecureAcceptAnything" for the reg if this allowed reg is set in the clusterimagepolicy
+				continue
+			}
```

Comment:
@mtrmac does this make sense? If the CIP scope is the same as the allowedregistries, only adds the signature policy not include the insecureAcceptAnything

Reply:
Yes, this looks correct to me.

Allowed = `example.com/foo` + policy from `example.com/foo` should be interpreted to allow that scope, but to enforce the policy, not to use `acceptAnything`.